src/transformers/models/vision_encoder_decoder/configuration_vision_encoder_decoder.py

```diff
@@ -15,8 +15,16 @@
 # limitations under the License.
 
 import copy
+from typing import Any, Mapping, Optional, OrderedDict
+
+from packaging import version
+
+from transformers.modeling_utils import PreTrainedModel
```

Comment:
Please use relative imports here to avoid circular dependencies :)

Reply:
Removed PreTrainedModel from imports. Instead used PretrainedConfig as arguments.